src/fragments/start/getting-started/ionic/data-model.mdx

```diff
@@ -116,41 +102,76 @@ Note: If you are using Angular 6 or above, you may need to add the following to
 };
 ```
 
-In your `src/app/app.component.ts` file, add the following imports and modifications to your class to to add data to your database with a mutation by using the `API.service` file which was generated when you ran `amplify add api`:
+First, enable the Angular forms modules in `src/app/app.module.ts`:
 
 ```javascript
-import { APIService } from './API.service';
+import { NgModule } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+import { RouteReuseStrategy } from '@angular/router';
+
+import { IonicModule, IonicRouteStrategy } from '@ionic/angular';
+
+import { AppComponent } from './app.component';
+import { AppRoutingModule } from './app-routing.module';
+import { AmplifyAuthenticatorModule } from '@aws-amplify/ui-angular';
+
+@NgModule({
+  declarations: [AppComponent],
+  entryComponents: [],
+  imports: [
+    BrowserModule,
+    IonicModule.forRoot(),
+    AppRoutingModule,
+    AmplifyAuthenticatorModule // UI Authenticator Module
+  ],
+  providers: [{ provide: RouteReuseStrategy, useClass: IonicRouteStrategy }],
+  bootstrap: [AppComponent]
+})
+export class AppModule {}
+```
 
-constructor(
-    //other variables
-    private apiService: APIService
-  ){
-    this.initializeApp();
-  }
+Then make sure to add the Authenticator styles into your `src/global.scss` file.
+
+```
+@import '~@aws-amplify/ui-angular/theme.css';
+```
+
+Next, in your `src/app/app.component.ts` file, add the following imports and modifications to your class to to add data to your database with a mutation by using the `API.service` file which was generated when you ran `amplify add api`:
+
+```javascript
+import { Component, OnInit } from '@angular/core';
+import { APIService, Todo } from './API.service';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: 'app.component.html',
+  styleUrls: ['app.component.scss'],
+})
+export class AppComponent implements OnInit {
+  public todos: Array<Todo>;
+
+  constructor(private apiService: APIService) {}
+
+  ngOnInit() {}
 
   createTodo() {
     this.apiService.CreateTodo({
-        name: 'ionic',
-        description: 'testing'
+      name: 'ionic',
+      description: 'testing',
     });
   }
+}
+
 ```
 
-Next, update `initializeApp()` in `src/app/app.component.ts` to list all items in the database by running a query on start and populating a local variable called `todos`:
+Next, update `ngOnInit()` in `src/app/app.component.ts` to list all items in the database by running a query on start and populating a local variable called `todos`:
 
 ```javascript
-export class AppComponent {
-  todos: Array<any>;
-
-  initializeApp() {
-    this.platform.ready().then(() => {
-      //other code
-      this.apiService.ListTodos().then((evt) => {
-        this.todos = evt.items;
-      });
+  ngOnInit() {
```

Comment:
I thought using `ngOninit` instead of calling a function from the constructor the more proper way of loading data when the app loads.

Reply:
Agreed, much cleaner!